pages/concepts/_meta.json

```diff
@@ -2,5 +2,6 @@
   "overview": "Key Concepts",
   "streaming-dataflow": "Streaming Dataflow",
   "efficiency": "Memory Effeciency",
+  "ha": "High Availability",
```

Comment:
For some reason this isn't showing up for me on the preview by default, but it does if I manually navigate to the ha page: https://readyset-public-docs-archived-git-ha-docs-readyset.vercel.app/concepts/ha

<img width="245" alt="image" src="https://github.com/readysettech/docs/assets/14879344/2a50f8cb-e71e-4472-89cc-1f9eafe23a74">

<img width="242" alt="image" src="https://github.com/readysettech/docs/assets/14879344/dd9e86c8-1304-4fb3-8e8b-1319637e9ede">

Reply:
shows up locally for me: 
![image](https://github.com/readysettech/docs/assets/1481027/06d2c451-a1b4-497e-9c3a-02ed0fc73c8d)
@daviddkkim is there anything else I need to do to get this to show up?

Reply:
Usually when you redeploy on vercel, it reflects the new nav bar changes. There's some weird caching thing happening i think. Let me kick one off for you!

Reply:
@glittershark @lukoktonos it seems to be reflected correctly after the redeploy - let me know what yall think!